api/src/main/java/io/jsonwebtoken/lang/UnknownClassException.java

```diff
@@ -48,16 +48,17 @@ public UnknownClassException(String message) {
     public UnknownClassException(Throwable cause) {
         super(cause);
     }
+    */
 
     /**
      * Constructs a new UnknownClassException.
      *
      * @param message the reason for the exception
      * @param cause   the underlying Throwable that caused this exception to be thrown.
-     *
+     */
```

Comment:
This diff looks funny, but this constructor was commented out, so I just uncommented it (and added a TODO to remove in the next version)